src/Strategy/Go/Types.hs

```diff
@@ -2,7 +2,7 @@ module Strategy.Go.Types
   ( GolangPackage -- don't export GolangPackage; export the smart constructor instead
   , mkGolangPackage
   , GolangGrapher
-  , GolangLabel(GolangLabelLocation) -- don't export GolangLabelVersion; export the smart constructor instead
+  , GolangLabel(..)
```

Comment:
It's necessary to expose the `GolangLabelVersion` constructor, which is needed in `buildGraph` because we don't use the `mkGolangVersion` smart constructor (to avoid using the buggy `fixVersion` implementation).